main.go

```diff
@@ -78,6 +78,11 @@ func main() {
 			Usage:  "git clone path",
 			EnvVar: "PLUGIN_WORKSPACE,DRONE_WORKSPACE",
 		},
+		cli.StringFlag{
+			Name:   "force-push",
+			Usage:  "git force push",
+			EnvVar: "PLUGIN_REMOTE_URL,DRONE_REMOTE_URL",
```

Comment:
I wasn't sure what to set this to, just copied a previous value.  This is my first attempt at contributing to drone plugin.

Reply:
Oh sorry didn't see this part.

The environment variable should be PLUGIN_FORCE_PUSH

Reply:
Done.